docs/book/content/contributing/contributor_guide.md

```diff
@@ -125,7 +125,7 @@ situations, in which case other contributors are here to help.
    this, run the following command from the command line from inside
    the OG-Core/ogcore directory:
     ```
-     OG-Core/ogcore$  pytest -m "not needs_puf and not regression"
+     OG-Core/ogcore$  pytest
     ```
    Note that running this full suite of tests may take close to 24 hours (depending on your hardware).  To run the small set of tests that run on each pull request (and take about 40 minutes), use  `pytest -m "not needs_puf and not local and not regression"`.  If the tests do not pass, try to fix the issue by using the information provided by the error message. If this isn't possible or doesn't work, the core maintainers are here to help via a [GitHub Issue](https://github.com/PSLmodels/OG-Core/issues).
```

Comment:
@jdebacker . You should update the text in line 130 of `contributor_guide.md` to reflect the lack of `needs_puf` and `regression` markers.